scripts/init-vscode

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# For people that want a base VSCode setup (with useful launchers etc)
+# But without checking in a .vscode directory by default: there are
+# people who definitely do not want it checked in.
```

Comment:
nit: can probably be simplified to describe the behavior. I had to do a double take to understand what I'm getting by running this.